server/data/timelineApiFilterOptions.ts

```diff
@@ -0,0 +1,30 @@
+export default class TimelineApiFilterOptions {
```

Comment:
This is the crux of how this is all going to work.
This new class `TimelineApiFilterOptions` is now a method arg to the API method.
The constructor is an optional map of optional flags; and it creates a property `queryParams` that has all the query string options with `false` defaults

The service method will new one of these up with the relevant filtering options and pass it into the API method; and the API method will use it for the `query` object.

In this PR the service method news up a `TimelineApiFilterOptions` but with no constructor arg/filter options, so the resultant filtering when we call the API is all the filter options set to `false` 👍